crates/extensions/c8y_mapper_ext/src/actor.rs

```diff
@@ -181,6 +181,9 @@ impl C8yMapperActor {
             self.process_registration_message(pending_entity.reg_message)
                 .await?;
 
+            // Wait for a while to allow the registration message to be processed by the cloud before sending the data messages
+            tokio::time::sleep(Duration::from_secs(1)).await;
```

Comment:
This feels like it's going to break if the bridge loses connection. Is there a particular reason why we can't do what we do for the main device, i.e. publish the registration message multiple times and wait until we receive a suitable `41` response to confirm the device definitely exists?

Reply:
Yeah, this is a known limitation. We're doing this low cost fix to just reduce the probability of this flakiness in 80-90% of the cases when everything is working fine. The plan was to try this  low cost fix and see if it reduces the flakiness considerably.

The other two approaches considered were:

1. Waiting for the duplicate device error over MQTT as done for the main device
2. Performing the entity creation over HTTP

Since both approaches fundamentally change the control flow within the mapper, with huge impact on all existing tests(esp unit tests) as well, we just decided to do this minimal fix for now. But, we can revisit these approaches later once we achieve some stability.